### PR TITLE
Syndicate Bundle Rebalance

### PR DESCRIPTION
**Pull request by Chronitonity**
First off, changes the price of the revolver ammo to halve it. You can make near infinite ammo at a hacked autolathe anyhow, its weird that it costs so much.

Rebalances the syndicate bundles, removes some of the terrible ones that no one used.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -3,19 +3,15 @@
 /obj/item/weapon/storage/box/syndicate/New()
 	..()
 	switch (pickweight(list("bloodyspai" = 1, "stealth" = 1, "bond" = 1, "screwed" = 1, "guns" = 1, "murder" = 1, "implant" = 1, "hacker" = 1, "lordsingulo" = 1, "darklord" = 1)))
```
Comment (TDcoolguy300) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: You dun fucked up, if you change or remove anything, update the switch here

Comment (Chronitonity) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: eeeeeeeugh

**Comment by Weisyl** [comment]
The psycho kit seems a bit underpowered to me. Also how about a ninja kit as well? Get like two boxes of throwing stars or something, maybe a katana too. Space ninja suit would be totally OP to give them though.

**Comment by danhibiki** [comment]
yesss

**Comment by Foosili** [comment]
I'm down

**Pull request merged**